scripts/Muon/GUI/Common/results_tab_widget/results_tab_presenter.py

```diff
@@ -78,11 +78,20 @@ def _on_new_fit_performed_impl(self):
         self._update_logs_view()
         self.view.set_output_results_button_enabled(True)
 
+    def _get_workspace_list(self):
+        fit_context = self.model._fit_context
+        workspace_list = []
+        fit_list = fit_context.fit_list
+        for ii in range(1, fit_context._number_of_fits + 1):
+            workspace_list.append(fit_list[len(fit_list) - ii].parameter_workspace_name)
```

Comment:
I think `fit_list[- ii].parameter_workspace_name` might do the same here as negative indexes count from the end.